app/models.py

```diff
@@ -20,5 +22,18 @@ class Facetname(str, Enum):
     nucleotide = "nucleotide"
     allergen = "allergen"
     trace = "trace"
-    
 
+    @staticmethod
+    def list():
+        return list(map(lambda c: c.value, Facetname))
```

Comment:
More pythonic + correct Facetname --> FacetName:

```suggestion
        return [c.value for c in  FacetName]
```